src/main/windows/html_email.ts

```diff
@@ -387,7 +395,24 @@ function makeBrowserView(
       if (url.startsWith('mailto:')) {
         open_url(url)
       } else {
-        shell.openExternal(url)
+        if (
+          url.startsWith('http:') ||
+          url.startsWith('https:') ||
+          url.startsWith('mailto:')
+        ) {
+          shell.openExternal(url)
```

Comment:
in  the bubbles, `mailto:` links are passed to  `processOpenQrUrl(link)` instead of `shell.openExternal()` - is that difference on purpose?

in general, bubble-view and full-view should behave quite similar from the user perspective - a link fit randomly to the bubbles or not, but they should act the same (it is a pity that we cannot use the same dialogs in both szenarios, btw)

Reply:
just a duplicated artefact that comes from copy pasting see 2 lines above. also `processOpenQrUrl` exist only in the frontend, so the `open_url` function is for forwarding the request to the frontend.